src/accelerate/utils/imports.py

```diff
@@ -219,6 +219,7 @@ def is_xpu_available(check_device=False):
     else:
         return False
 
+    import intel_extension_for_pytorch
```

Comment:
This needs to be in a try/except block.

Reply:
thanks for the review, I just finished the format error, as for try/block, I think we have no need to add a try block for "import  intel_extension_for_pytorch", because in there is check in "is_ipex_available()", if is_ipex_available is true, "import intel_extension_for_pytorch" will always be okay.

Reply:
> This import is not under that test/

what does test/ you mean? if you meant I should put it into the check block, I don't think so, when check=False(actually, in this repo, I don't see any caller set it as True), you will go to  line 231 "hasattr(torch, "xpu") and torch.xpu.is_available()", if you don't  import intel_extension_for_pytorch, line 231 will always return False, which is unexpected.

Reply:
The import `import intel_extension_for_pytorch` is not under anything testing that the library is actually available. So this will make the code break for anyone who doesn't have it installed.

Reply:
> The import `import intel_extension_for_pytorch` is not under anything testing that the library is actually available. So this will make the code break for anyone who doesn't have it installed.
thanks for the comment. But, 
in line 217(if is_ipex_available()), it will check whether intel_extension_for_pytorch is installed or not, if it is not installed, it will go to line 223, that is "return False" directly.

Reply:
Oh, sorry. I don't know how I missed that else.